src/main/scala/cromwell/binding/Scope.scala

```diff
@@ -1,14 +1,168 @@
 package cromwell.binding
 
+import cromwell.binding.AstTools.AstNodeName
+import cromwell.parser.WdlParser.{AstList, Ast, AstNode}
+import scala.collection.JavaConverters._
+
+
+object Scope {
+
+  /**
+   * Utility class to generate a ScopeTree from an Ast.
+   */
+  private class ScopeTreeGenerator {
+
+    /* Keep track of the # of created Scatter nodes. */
+    var scatterCount: Int = 0
+
+  /*
+  * Retrieve the list of AstNodes from the AST body.
+  * Return empty seq if body is null or is not a list.
+  */
```

Comment:
Formatting is slightly off for this comment block... Try to use `/**` comment blocks instead, and IntelliJ will auto-format